services/SessaoTreinoService.js

```diff
@@ -118,6 +118,19 @@ module.exports = {
 		});
 	},
 
+	// Serviço para incrementar o treino concluidos do utilizador
+	treinoConcluidoUtilizador: (id_utilizador, n) => {
+		return new Promise((aceito, rejeitado) => {
+			db.query('Update Utilizadores SET treinos_concluidos = ? WHERE id = ?', [n, id_utilizador], (error, results) => {
+				if(error){
+					rejeitado(error);
+					return;
+				}
+				aceito(results);
+			})
+		})
+	},
```

Comment:
Após a sessão de treino for concluída, o campo treinos_concluidos do utilizador vai ser auto incrementada